content/post/yet-another-reason-to-privatize-the-police.md

```diff
@@ -34,21 +34,21 @@ This brings us to the second major problem with the state-funded police: the pro
 
 Competition gives rise to the abundance of different products and services available to us. It encourages innovation, cost-cutting, day-to-day efficiency, and friendly customer service. Business that do a better job of implementing these things are going to satisfy a greater number of consumers and be rewarded with larger revenue and increased profits. The moment consumers become dissatisfied with a business for any reason, they file complaints, leave negative reviews, request refunds and returns, or take their money elsewhere. As a business owner or entrepreneur, refusing to correct any inefficiencies in your business, ignoring customer feedback, or ceasing to innovate is an open invitation for another business or entrepreneur to take your place. Because markets operate in this manner, they present us with a tremendous overflow of choices that allow us to seamlessly switch between products and services to appease our ever-changing preferences. 
 
-But rather than subjecting state-funded police to the same mechanisms that make market economies so competitive and flowing with choice &mdash; freedom of entry, the profit and loss mechanism, competitive innovation, and the desire to reduce business costs &mdash; there is no question that a majority of our nation would still have it be the sacrosanct duty of the state alone to provide us with policing services. And why should that be? Policing services are no more unique than any other service, nor is their provision too complicated for the world's most technologically advanced and innovative private sector[^4]. Why have a one-size-fits-all, forcibly funded monopoly on police provision as opposed to tens, hundreds, or thousands of voluntarily funded, private police firms that are actually beholden to their customers and tied to the profit and loss system of the marketplace?
+But rather than subjecting state-funded police to the same mechanisms that make market economies so competitive and flowing with choice &mdash; freedom of entry, the profit and loss mechanism, competitive innovation, and the desire to reduce business costs &mdash; there is no question that a majority of our nation would rather it be the sacrosanct duty of the state alone to provide us with policing services. And why should that be? Policing services are no more unique than any other service, nor is their provision too complicated for the world's most technologically advanced and innovative private sector[^4]. Why have a one-size-fits-all, forcibly funded monopoly on police provision as opposed to tens, hundreds, or thousands of voluntarily funded, private policing firms that are actually beholden to their customers and tied to the profit and loss system of the marketplace?
```

Comment:
```suggestion
But rather than subjecting state-funded police to the same mechanisms that make market economies so competitive and flowing with choice &mdash; freedom of entry, the profit and loss mechanism, competitive innovation, and the desire to reduce business costs &mdash; there is no question that a majority of our nation would rather it be the sacrosanct duty of the state alone to provide us with policing services. And why should that be? Policing services are no more unique than any other service, nor is their provision too complicated for the world's most technologically advanced and innovative private sector[^4]. Why have a one-size-fits-all, forcibly funded monopoly on police provision as opposed to tens, hundreds, or thousands of voluntarily funded, private policing firms that are actually beholden to their customers and subject to the profit and loss system of the marketplace?
```